tests/execution_engine/test_sqlalchemy_execution_engine.py

```diff
@@ -732,3 +736,46 @@ def validate_tmp_tables():
     )
 
     validate_tmp_tables()
+
+
+def test_sample_using_random(sqlite_view_engine, test_df):
+    my_execution_engine: SqlAlchemyExecutionEngine = SqlAlchemyExecutionEngine(
+        engine=sqlite_view_engine
+    )
+    test_df.to_sql("test_table_0", con=my_execution_engine.engine)
+
+    p: float = 2.0e-1
+    batch_spec: SqlAlchemyDatasourceBatchSpec = SqlAlchemyDatasourceBatchSpec(
+        table_name="test_table_0",
+        schema_name="main",
+        sampling_method="_sample_using_random",
+        sampling_kwargs={"p": p},
+    )
```

Comment:
Thank you for finding and fixing this @alexsherstinsky , just wanted to request the `p: float = 1.0` test that we described during our Zoom chat 🙇🏼 🙇🏼🙇🏼 otherwise LGTM

Reply:
@Shinnnyshinshin Done!